feat(content): Successors: Ghosts I (#12221)
endless-sky/endless-sky: data/successors/successor 2 ghosts.txt

			choice
				`	"Could you help me find Saajret, then?"`
					goto saajret
				`	"I tried to land there and they told me to seek authorization."`
					to display
						has "successors: ghosts ally referred"
					goto referred
				`	(Leave and go about your business.)`
					decline

			label referred
			`	"That I cannot give," they say, "nor would I do so if empowered."`
			choice
				`	"Could you help me find Saajret, then?"`
					goto saajret
				`	(Leave and go about your business.)`
					decline

			label saajret
			`	The concierge makes a noise like humming, but lower and more musical. "Perhaps. A friend, or companion of yours? But I require more than their third name, traveler."`
			choice
				`	"Saajret, of Somber Sejra."`
					goto "saajret of somber sejra"
				`	"I don't know them by any other names."`
			`	"A rose! A rose," they say, "by any other name would smell as sweet. Not so close a friend, I suppose. But I suppose too much, methinks."`
			choice
				`	"Wait! What about 'Saajret, of Somber Sejra?'"`
					goto "saajret of somber sejra"
				`	(Leave and go about your business.)`
					decline

			label "saajret of somber sejra"

> 			`	"Ah." Their skin shifts to a deep pink, but only for a moment. They pull a long column of metal from the panel in front of them, examining it closely for a time before returning their attention to you. "If thou know'th them well enough to speak of, then 'tis sure thou can'st appreciate the stresses on their time. But," they say, glittering, "I shall notify them of thy desire to meet, and I shall give them thy name and this," they say producing a data-sphere of the sort you've seen the Successors use before. "And if they wish to engage thee, then they may."`

Reviewer: ```suggestion
			`	"Ah." Their skin shifts to a deep pink, but only for a moment. They pull a long column of metal from the panel in front of them, examining it closely for a time before returning their attention to you. "If thou know'th them well enough to speak of, then 'tis sure thou canst appreciate the stresses on their time. But," they say, glittering, "I shall notify them of thy desire to meet, and I shall give them thy name and this," they say producing a data-sphere of the sort you've seen the Successors use before. "And if they wish to engage thee, then they may."`
```

			`	'Tis good of thee, captain, that thou sought me out upon learning of this recent incident. Thine instincts do thee justice. Nonetheless, I've naught to give thee. The matter underway is of Kaatrij and does not pertain to thee nor the other Houses. If thou well-regards this House, keep clear of this. Enjoy our fruitful world, assist our grateful citizens, be regaled by what gifts we yet may give. Do not pry. This I beseech you, for our sake and for thine: do not be tempted by our secrets' jaws.`
			`	This I myself attest as K- I- Saajret, of Somber Sejra.`
				decline

# Seineq investigation

mission "Successors: Ghosts 1"
	landing
	name "Assist House Seineq"
	source
		attributes "successor"
		not attributes "uninhabited"
		not planet "Shassa-Wyra-Orrou"
	destination "Shassa-Wyra-Orrou"
	description "House Seineq would like to meet you at <destination>."
	to offer
		has "Successors: Ghosts Prelude Incident: offered"
		random < 50
	on offer
		conversation
			branch raaqa-puan-uuoru
				has "flagship planet: Raaqa-Puan-Uuoru"

> 			`Soon after you exit your ship on this planet, you are discreetly pulled aside by a trio of Successor military officers. Their leader gives you a gentle bow. "Honored captain," they say, "We've come to inform thee that the noble House Seineq requests thy presence for the purposes of lending aid to their ally, the wounded House Kaatrij."`

Reviewer 2: Do these aliens bow? I think I vaguely remember they use some other physical action to express their emotions and stance, colors of course and some other body movements? Need to double-check I guess.

			`	"Captain, captain, we lament thy wait. I am Sijra and this," they say, gesturing to the other, "is Rijatet." Rijatet offers a short splay of the arms on hearing their name. "We represent House Seineq here."`
			choice
				`	"What do you want from me?"`
					goto task
				`	"Did Seineq establish all this?"`
				`	"What is this place?"`
			`	Sijra nods as colors swim over their body. "A temporary post, approv'd by the New's combined navy. In times as these, 'tis a boon to watch over these oft-forgotten systems."`
			`	Rijatet interjects. "'Tisn't why thou'rt here, though." Sijra hums gently at this, and they slither under one of the mist generators, writhing a little as if to get comfortable.`
			label task
			`	"Our ally, Kaatrij," says Sijra, "has ceased their internal messages to us, and the rest of the New. But even if they do not wish to share, we can derive our own answers well enough."`
			`	"This was an attack," says Rijatet. Their voice is low and rumbly, like rocks rolling down a hill. "Kaatrij would not close up otherwise. They do not trust the other Houses. They never have. They think another House has wounded them."`

> 			`	"And thou, captain, thou'rt a... free element. Thy place in this can't be eas'ly divined. We can handle one alien, and their host, but cans't be sure thyself is all we have to reckon with." Waves of color run over Sijra's body, and one of their legs twitches gently against the floor.`

Reviewer: Cans't isn't quite correct I think.

```suggestion
			`	"And thou, captain, thou'rt a... free element. Thy place in this can't be eas'ly divined. We can handle one alien, and their host, but canst not be sure thyself is all we have to reckon with." Waves of color run over Sijra's body, and one of their legs twitches gently against the floor.`
```

				goto decision

			label spy
			`	A faint pink color forms at the tips of Sirja's arms, though what exactly this connotes you cannot tell. "No, Captain. That is not thy art, methinks. We only ask thee to be attentive in thy duties, and to tell us of anything untoward."`
			label decision
			choice
				`	"Very well."`
					goto yes
				`	"I've had enough of this. No."`
			`	The two Successors eye you, a shade of blue slowly creeping over their skin. "Disappointing, Captain. But it is thy own decision."`
				decline

			label yes
			action
				log "Accepted an offer by House Seineq to report back my dealings with the other Houses, and whether I observe anything incriminating."
			`	"Excellent!" says Rijtatet. "In that case, Sijra and I have nothing further for thee. We shall call thee back when we are ready for thy report."`
				decline

mission "Successors: Ghosts Mark Houses"
	entering
	destination "Shassa-Wyra-Orrou"
	name "Investigate the other High Houses"
	description "House Seineq is interested in what tasks the other Houses might offer a free agent. Their homeworlds would be a good place to find work for them."
	to offer

> 		has "Successors: Ghosts 4: declined"

Reviewer 3: This can still offer even if you refused to report on other Houses to Seineq. Is that intentional?

			choice
				`	"Sure, I'll take you."`
					goto helped
				`	"Sorry, you'll have to figure something else out."`
			`	Saajret struggles to keep their disappointment from showing. "Very well," they manage to say before leaving.`
				goto end

			label helped
			action
				set "Successors: Ghosts Helped Saajret"
			`	Saajret keeps their composure for a second before a faint pink color creeps up their face and arms. "My thanks to thee, Captain," they say. "The world is Osolaa-Val-Mavra; I'll give thee its place."`

			label end
			`	With nothing left to do here, you prepare to leave.`

> mission "Successors: Ghosts 7a"

Reviewer 2: Hum. If I reload the game on this day before accepting the mission, I won't see this being offered at all.

I don't know if that's a problem story-wise or not..

			`	"What is this place?" you ask.`
			`	A blue tone makes its way over Saajret's face. "'Twas an arc of homes and shops before they were abandoned," they say. A couple of snowflakes land on their skin and their arms wriggle against the white ground. "Thou may'st join me if thou desirith. Kaatrij demands no secrecy from us here."`
			choice
				`	(Go with them.)`
				`	(Stay.)`
					goto stay
			`	The two of you make your way through the abandoned streets. You are struck by how well-preserved everything is; more than those of any other Predecessor world, the buildings here seem only temporarily vacant, as if the two of you had merely caught the city on a day with everyone missing. A trio of little cleaning robots run around the rim of a fountain, polishing away the snow before it even has time to melt in the sun.`

> 			`	Saajret follows their own path, periodically checking what must be a map on a datapad they carry. Occasionally their skin will erupt in color as they seey something that interests them, but they never stay distracted for long. Eventually the two of you arrive in an elegant square of organic buildings; Saajret checks their map a few times before declaring, "here we are."`

Reviewer 2: > seey

"see"?

			`	Not long after you see this, a thin sandstorm starts to roll in, and you take the <ship> back up to orbit before the mysterious absence of airborne grit gives your ship away.`
				flee

	on decline
		log "Approached the base on Qasa-Sija-Iri while under cloak. Its design vaguely resembles Quarg construction - perhaps they have some answers about what's been going on with the Successors."

mission "Successors: Ghosts 9"
	landing
	source
		attributes "human quarg" "hai quarg"
	to offer
		has "successors: ghosts seen quarg"
	on offer
		conversation
			`Do you want to ask the Quarg here about their base on the Successor moon Qasa-Sija-Iri?`
			choice
				`	(Yes.)`
				`	(Not now.)`
					defer
			`	Though so far individual Quarg have seemed equivalent in their ability to answer your questions, it is several minutes before you spot one here who, for some reason, seems like it would be able to answer. When you repeat your question to it, it blinks slowly, and answers even more so.`

> 			`	"I see," it says. "You have uncovered one our great secrets of the galaxy. You were right to think that the world you speak of is ours. After what that foolish species did to themselves, we the Quarg could not bear to leave them unobserved. But we dared not do so unshrouded, lest we once again rouse them to the heights of envy - so we built no ringworld, no stations, no open worlds. Only a little... watching post, to keep our gaze over the Myiara."`

Reviewer 4: Yep, this should do it.
```suggestion
			`	"I see," it says. "You have uncovered one of our great secrets of the galaxy. You were right to think that the world you speak of is ours. After what that foolish species did to themselves, we the Quarg could not bear to leave them unobserved. But we dared not do so unshrouded, lest we once again rouse them to the heights of envy - so we built no ringworld, no stations, no open worlds. Only a little... watching post, to keep our gaze over the Myiara."`
```

			`	You interject. "The Myiara? Do you mean the Successors?"`
			`	The Quarg considers the term. "Before they called themselves that, they were the Myiara. Have they truly become so rooted in their memories of their valorous past?" It makes a noise like the rumbling of distant thunder. "You must have questions."`
			choice

> 				`	"What is it that they did to themselves? Did they build some kind of terrible weapons?"`

Reviewer 3: ```suggestion
				`	"What is it that they did to themselves? Did they build some kind of terrible weapon?"`
```

Reviewer 3: Sorry, this was a grammar suggestion, didn't mean to change the meaning of the sentence. How about:
```suggestion
				`	"What is it that they did to themselves? Did they build terrible weapons?"`
```

			label interrogation
			`	With apparently no choice, you accompany the officers back to their ship; they remove your sidearm and take you to a room inside. Two stand guard over you while the rest attend to other things outside, and it's only after the better part of an hour that someone else finally enters, a tall Successor who looks over you with an incisive glare.`
			`	"Captain <first> <last>," they say. "How did'st thou acquire cloaking technology?"`
			choice
				`	"It comes from another society, far away from here."`
				`	"I was given it by a human government; they found it in an alien cargo hold."`
					to display
						has "outfit (all): Cloaking Device"
			`	"So it was not given to thee by a faction here, among the Successors?"`
			`	"No," you say. The Successor relaxes, if only a little. "We will search thy ship t'confirm this," they say. "But I do not believe thou lieth."`
			branch "cloaks"
				has "flagship attribute: cloak"

			choice
				`	"You won't find anything, my current ship isn't cloak-capable."`
			`	The Successor looks over you with a curious gaze. "Intriguing," they say. "'Tis no matter; we shall consult our records of thy craft."`

			label cloaks

> 			`	After a moment passes, your interrogator leans over you and the guards in the back stand, attentive. "Now we must decide what is to be done with thee," they say. "Now that thou hast learned of our arrangement with the Quarg. Thou should'st understand we do not wish that information to be avail'ble to all. Not all the Houses nor their members regard the Quarg with amity."`

Reviewer 3: ```suggestion
			`	After a moment passes, your interrogator leans over you and the guards in the back stand, attentive. "Now we must decide what is to be done with thee," they say, "now that thou hast learned of our arrangement with the Quarg. Thou should'st understand we do not wish that information to be avail'ble to all. Not all the Houses nor their members regard the Quarg with amity."`
```

			choice
				`	"All right, I accept."`
					goto yes
				`	"No."`
					goto no

			label yes
			action
				set "successors kaatrij vassal"
				event "successors: ghosts unrestrict"
				"reputation: House Kaatrij" >?= 2
			`	"Excellent," the Successor says. "I would have thee come with us, then, to the moon where this all began, our base on <planet>. There, some of thy no-doubt-pressing questions shall be answered."`
				accept

			label no
			branch "was discreet"
				has "Successors: Ghosts: discreet"

			`	"A shame," your interrogator says, making a motion with one of their arms; behind them, one of the guards raises their rifle. Instinctually, you go to grab your own sidearm, only to remember that it was taken from you minutes ago just as the guard's railgun round pierces your body.`
				die

			label "was discreet"
			action

> 				set "successors free agent" # This doesn't do anything now, but if Ghosts is ever expanded to let the player character ally with other Houses, this tag would be useful to have.

Author: I've added a diegetic "are you sure" choice as an alternate way of addressing this; unlike other points where the player could fail because they lost a fight or didn't realize what a mission entailed, I want this to be a clear _choice_ that the player makes knowing that nothing comes after it if they decline. Thus no dialog.

						not "Successors: drone"
					goto drone
				`	"How much cloaking technology do you actually have?"`
					to display
						not "Successors: cloak"
						has "Successors: attack"
					goto cloak

			label attack
			action
				set "Successors: attack"
			`	"'Twas one of the High Houses, but which exactly, we do not know yet for certain." Kampa contemplates for a moment. "Each of the Houses keeps a few heirloom warships from the time of the old empire, and most dare not bring them from the gilded vaults where they lie, for they are far too precious to risk in petty conflicts. Yet another still fielded one of theirs, raided the moon, killed hundreds, all to steal a single cloaking unit. Now they flit about unseen."`
				goto questions

			label seineq
			action
				set "Successors: seineq"
			`	There's a pause as Kampa looks at you with incisive eyes. "Thou'rt fair to ask it," they say. "No," they say, "'twas not Kaatrij that struck them. Once our cloak was stolen, our 'lusive foe had all 'twas needed t'frame us of that trait'rous blow: t'let themselves been seen with signs that we alone had once posses'd."`
				goto questions

			label drone
			action
				set "Successors: drone"

> 			`	Kampa's skin ripples with hollow mirth. "Not exactly," they say. "'Twere creations of an ally of the old House Kaatrij during the last great inter-Houses war, a bold engineer of Aqrabe. Autonomous as they were, they could - when stripp'd of 'twas most noxious to our victory - remain to ward off plund'rers of our cities and hidden vaults."`

Reviewer 3: `when stripp'd of 'twas most noxious to our victory` Not sure what this means?

				flee
			label fragile
			`	Your ship's shields go down after the first few shots and it doesn't even take the full volley to destroy your vessel entirely.`
				die

			label ally
			`	"Thou art denied, for thou'rt without a writ of authorization. Converse with thy superior if this is in error, and return only when thou canst present it. Now return to orbit, on pain of death, ally or no."`
			action
				set "successors: ghosts ally referred"
				log "Attempted to land on Qasa-Sija-Iri but was denied for lack of authorization. Instructed to converse with a superior to acquire authorization to land."
			choice
				`	(Return to orbit.)`
					flee
				`	(Continue anyway.)`
					goto proceed

			label elsewhere
			`	"Irrelevant. No civilian vessels are permitted here. Return to orbit, on pain of death."`
			choice
				`	(Return to orbit.)`
					flee
				`	(Continue anyway.)`
					goto proceed

mission "Successors: Ghosts QSI 2"
	priority
	source
		government "House Kaatrij"
	to offer
		has "successors kaatrij patron"
		not "successors: ghosts restriction lifted"
	on offer
		conversation

> 			`Though the Kaatrij spaceport is usually quiet, its patrons speaking in hushed whispers, today it is utterly soundless. The very few Successors who go about their business inside do so without a word and stick to only the most thinly-hydrated pathways such that they leave only the most minute of splashes in their wake.`

Reviewer 3: Trying to reduce repetition of "only the most"
```suggestion
			`Though the Kaatrij spaceport is usually quiet, its patrons speaking in hushed whispers, today it is utterly soundless. The very few Successors who go about their business inside do so without a word and stick to only the most thinly-hydrated pathways such that they leave only the smallest of splashes in their wake.`
```

			`	"And of course Seineq means to make this worth your while," says Rijatet. "Thou shalt be compensated with a fee of <payment> upon completion. We are not at liberty to negotiate this."`
				accept

	on enter "Kasii-Sola"
		dialog `You release one of House Seineq's scanners, which orients itself with small thrusters before nestling into low planetary orbit.`
	on enter "Khasola-Ryuit"
		dialog `You eject one of the scanners into space here at the northernmost reach of the Predecessors' region.`
	on enter "Eta Carinae"
		dialog `You release one of the scanners from your cargo hold, letting it disappear in the glare of this system's enormous star.`
	on enter "Avasaa-Novaa"
		dialog `You eject one of the scanners, which proceeds into a close solar orbit. Perhaps the gravity well helps shield it from the destructive spatial phenomena here.`

	on visit
		dialog phrase "generic waypoint on visit"

	on complete
		conversation

> 			`Docking control at the <planet> spaceport directs the <ship> to a protected, fabric-shrouded hangar on the western side of the new facility. A pair of Successor ground crew supervise the automated refueling and maintenance processes, and one of them waves you down from the cockpit, where they hand you a data-sphere. A few seconds after it touches your skin, it plays a message from Rijatet:`

Reviewer 3: ```suggestion
			`Docking control at the <planet> spaceport directs the <ship> to a protected, fabric-shrouded hangar on the western side of the new facility. A pair of Successor ground crews supervise the automated refueling and maintenance processes, and one of them waves you down from the cockpit, where they hand you a data-sphere. A few seconds after it touches your skin, it plays a message from Rijatet:`
```

> 			`	"Our thanks to thee for setting the sensors, Captain. We saw their first reports only shortly after as thy own craft jumped away, and nothing since. We shall remain as watchful as we ought to be. For thine efforts, Seineq's credited thee the agreed payment of <payment>. If thou wishest for further, join us at the port here tonight for another task."`

Reviewer 3: ```suggestion
			`	"Our thanks to thee for setting the sensors, Captain. We saw their first reports only shortly after thy own craft jumped away, and nothing since. We shall remain as watchful as we ought to be. For thine efforts, Seineq's credited thee the agreed payment of <payment>. If thou wishest for further, join us at the port here tonight for another task."`
```

			`	The Successors around the screen murmur and slither around each other. As the broadcast goes on and the crowd grows and the first casualty numbers begin to come in, the overlapping translated speech in your ears starts to smear together; you find yourself stumbling against a wall in the spaceport, only able to get up once you've momentarily plucked the translation device from your jacket.`
				decline

mission "Successors: Ghosts Seineq Memorial"
	invisible
	source "Yoqqa-Vasa-Vasa"
	to offer
		has "event: successors: ghosts seineq memorial"
	on offer
		conversation
			`While exploring House Seineq's spaceport here on <origin>, you take notice of a series of obelisks erected in the middle of one of the concourses; they weren't here the last time you visited. As you draw nearer, you can make out an inscription on a plaque that stands before them:`
			`	"In remembrance and grief we stand,`
			`	reminders of the deaths of the`
			`	eighteen killed in service of`
			`	our noble House off'ring succor`
			`	to its ally House Kaatrij."`
			choice
				`	(Pay your respects.)`
				`	(Walk on.)`
					decline

> 			`	You offer the memorial a few moments of quiet contemplation. On each of the pillars a name is engraved; though most you do not recognize, one is labeled "Asirij T- Rijatet."`

Reviewer 3: Is the player able to read the Successor script and recognize the name here?

Author: There are a couple of other instances where it's established that the Successor translation device works on text as well. Whether the player character recognizes the name as that of one of the officers in the brief Seineq missions is up to the *actual player*, I suppose.

				decline

# Suspicion

mission "Successors: Ghosts 5"
	landing
	clearance
	name "Transport equipment for Kaatrij"
	description "Some scientific equipment belonging to House Kaatrij needs to return to <destination> by <date> as the House consolidates itself after its recent crises."
	deadline
	source
		government "Successor" "New Houses" # So, not the homeworlds of any of the other houses.
	destination "Raaqa-Puan-Uuoru"
	to offer
		or
			"successors: ghosts suspicion" >= 4
			"successors: ghosts houses dealt with" >= 2
		has "event: successors: ghosts seineq attacked"
	cargo "Scientific equipment" 10
	passengers 3
	on offer
		conversation
			branch kaatrij
				has "successors kaatrij patron"

> 			`Waiting in the spaceport here as you arrive are a trio of Successors; two guards and one other individual who wears a black, cocoon-like outfit with lots of pockets. Both of the guards bear the insignia of House Kaatrij.`

Reviewer: ```suggestion
			`Waiting in the spaceport here as you arrive are a trio of Successors, two guards and one other individual who wears a black, cocoon-like outfit with lots of pockets. Both of the guards bear the insignia of House Kaatrij.`
```

Reviewer 4: I think this one was accurate before.

Reviewer 5: I agree. Using a comma makes it seem like a list, so it appears as though there are three successors in addition to two guards and somebody else.

Reviewer: Indeed, yeah. Oops. I was going by "semicolon should be replaceable by period", but it seems that semicolons should still be used if there's a list with commas afterwards. Thanks!

> 			`	The black-draped individual bows gently to you as green sparkles dot their skin. "Captain <first> <last>?" they say. When you indicate affirmatively, they continue. "I am Saajret, of Somber Sejra, an envoy of the noble scions of House Kaatrij. Kaatrij hopes that thou might'st do us a boon."`

Reviewer: ```suggestion
			`	The black-draped individual bows gently to you as green sparkles dot their skin. "Captain <first> <last>?" they say. When you indicate affirmatively, they continue. "I am Saajret, of Somber Sejra, an envoy of the noble scions of House Kaatrij. Kaatrij hopes that thou mightst do us a boon."`
```

				goto questions

			label kaatrij

> 			`Waiting in spaceport here as you arrive are a trio of Successors; two guards and one other individual who you recognize as Saajret, of Somber Sejra. Kaatrij's archivist looks different from the last time you saw them; their outfit is wrapped up more tightly and the colors of their face seem muted and subdued.`

Reviewer: ```suggestion
			`Waiting in the spaceport here as you arrive are a trio of Successors, two guards and one other individual who you recognize as Saajret, of Somber Sejra. Kaatrij's archivist looks different from the last time you saw them; their outfit is wrapped up more tightly and the colors of their face seem muted and subdued.`
```

			`	Saajret bows to you gently, green and reddish tones swirling on their skin. "Captain <first> <last>," they say, "whose patience I have tested thoroughly, it would seem. Kaatrij sends me to ask of thee a boon to us."`

			label questions
			choice
				`	"Why haven't I heard from you since the incident?"`
					to display
						has "successors kaatrij patron"
					goto silence
				`	"What sort of 'boon?'"`
					goto boon
				`	"Why me?"`

			label me

> 			`	"Thou'rt earning thyself a reputation for efficiency and discretion," Saajret says. "And Kaatrij has need of both. Some scientific equipment here must return to the homeworld, ideally with haste. Can'st thy craft serve?"`

Reviewer: ```suggestion
			`	"Thou'rt earning thyself a reputation for efficiency and discretion," Saajret says. "And Kaatrij has need of both. Some scientific equipment here must return to the homeworld, ideally with haste. Canst thy craft serve?"`
```

				goto offer

			label boon

> 			`	"Thy reputation for professionalism seems well-earned, Captain," they say. "Some scientific equipment stored here must return to the homeworld with haste. Can'st thy craft serve?"`

Reviewer: ```suggestion
			`	"Thy reputation for professionalism seems well-earned, Captain," they say. "Some scientific equipment stored here must return to the homeworld with haste. Canst thy craft serve?"`
```

				goto offer

			label silence

> 			`	Saajret hums, colors continuing to swim over their body. Eventually, they speak. "In the wake of Kaatrij's wound, our suspicion of others was great and wide. And thou were't included in this, for these incidents came only soon after thy arrival. Some believe thou play'd some part in this, some do not." They pause for a moment, running their eyes over you. "'Tis the latter faction that offers thee this opportunity: some equipment here must return to the homeworld. A chance to demonstrate thy benevolence, mayhaps?"`

Reviewer: Were't is a contraction of "were it".

```suggestion
			`	Saajret hums, colors continuing to swim over their body. Eventually, they speak. "In the wake of Kaatrij's wound, our suspicion of others was great and wide. And thou wert included in this, for these incidents came only soon after thy arrival. Some believe thou play'd some part in this, some do not." They pause for a moment, running their eyes over you. "'Tis the latter faction that offers thee this opportunity: some equipment here must return to the homeworld. A chance to demonstrate thy benevolence, mayhaps?"`
```

			label offer
			choice
				`	"All right. I assume the House is paying?"`
					goto accepted
				`	"I don't want to get involved in this."`

			`	"Very well," Saajret says. "'Tis thy own choice, after all."`
				decline

			label accepted
			`	"Of course," Saajret says. "I will see the items are loaded forthwith."`
				accept

	on decline
		dialog "successor ghosts story failed"

	on fail
		dialog "successor ghosts story failed"

	on complete
		payment 40000
		conversation
			branch "visited before"
				or
					has "successors kaatrij patron"
					has "visited planet: Raaqa-Puan-Uuoru"

> 			`You've heard <planet> described by outsiders as an insular and well-guarded world, but neither of those prepared you for how strikingly quiet it is. The rainy spaceport seems almost deserted, the loudest sound almost always the pattering of heavy drops on the sail-like projections overhead, only occasionally interrupted by a soft-spoken announcement about an incoming ship landing or departure. None of the Successors you transported give any indication about whether this is typical or just a result of Kaatrij's recent incidents.`

Reviewer 3: ```suggestion
			`You've heard <planet> described by outsiders as an insular and well-guarded world, but neither of those prepared you for how strikingly quiet it is. The rainy spaceport seems almost deserted, the loudest sound almost always the pattering of heavy drops on the sail-like projections overhead, only occasionally interrupted by a soft-spoken announcement about an incoming ship landing or departing. None of the Successors you transported give any indication about whether this is typical or just a result of Kaatrij's recent incidents.`
```

				goto paid

			label "visited before"

> 			`The <planet> spaceport is caught in the midst of a gentle rainstorm and the loudest sound you hear during the time you spend there is the patter of the heavy drops overhead and the occasional whirring of the machinery the dockworkers use to unload your cargo.`

Reviewer 3: ```suggestion
			`The <planet> spaceport is caught in the midst of a gentle rainstorm. The loudest sound you hear during the time you spend there is the patter of the heavy drops overhead and the occasional whirring of the machinery the dockworkers use to unload your cargo.`
```

			label paid
			`	Saajret meets with you on the landing pad, informing you that you've been credited <payment> by the House for your services. Before leaving, they ask you to return to the spaceport this evening to meet with them.`

mission "Successors: Ghosts 6"
	name "Visit <planet> again"
	description "Saajret, of Somber Sejra has been instructed by House Kaatrij to scan the wildlife on <destination>, which must be of interest to multiple of the High Houses."
	source "Raaqa-Puan-Uuoru"
	destination "Raaqa-Uur-Kaav"
	passengers 1
	to offer
		has "Successors: Ghosts 5: done"
	on offer
		conversation
			`The black-shrouded Saajret meets you back at the spaceport as the sun is setting, joining you in a lobby area looking out on a landing pad for Kaatrij's military ships. As they start to speak, a pair of Veusaden hover to the ground in the distance, their repulsor thrusters kicking into the air drops of the everpresent rain.`

> 			`	"I've managed to convince Kaatrij that thou might be helpful to us in one more task," they say. "As the situation develops, they now agree thou'rt too substantive to deny all dialog. Dost thou remember thy prior visit to <planet>, when thou faced the Kijran?"`

Reviewer: ```suggestion
			`	"I've managed to convince Kaatrij that thou mightst be helpful to us in one more task," they say. "As the situation develops, they now agree thou'rt too substantive to deny all dialog. Dost thou remember thy prior visit to <planet>, when thou facedst the Kijran?"`
```

			choice
				`	"Yes."`
				`	"It was hard to forget."`
			branch kaatrij
				has "successors kaatrij patron"

			action
				log "People" "Saajret, of Somber Sejra" "Saajret, of Somber Sejra is an archivist, among other things, for the Successor High House Kaatrij. They are a member of the noble lineage of the now-defunct House Sejra, some of whose citizens were absorbed into House Kaatrij during the partial collapse of their Predecessors' society."

> 			`	"My role here is... something of an archivist," Saajret says, "collecting and ordering information, items, records, of our Predecessors 'fore we left that period behind. Kaatrij believes certain data about the creatures on the old homeworld would be of use to them, and since thou now provides them the opportunity," they say, spreading their arms in a wide gesture, "Kaatrij is willing to compensate thee for taking me there to scan them."`

Reviewer: ```suggestion
			`	"My role here is... something of an archivist," Saajret says, "collecting and ordering information, items, records, of our Predecessors 'fore we left that period behind. Kaatrij believes certain data about the creatures on the old homeworld would be of use to them, and since thou now providest them the opportunity," they say, spreading their arms in a wide gesture, "Kaatrij is willing to compensate thee for taking me there to scan them."`
```

				goto choices

			label kaatrij

> 			`	"I'm an archivist, as I told thee before," Saajret continues. "Kaatrij believes certain data about the creatures there would be of use to them and, since thou now provides them the opportunity, shall compensate thee for taking me there to scan them."`

Reviewer: ```suggestion
			`	"I'm an archivist, as I told thee before," Saajret continues. "Kaatrij believes certain data about the creatures there would be of use to them and, since thou now providest them the opportunity, shall compensate thee for taking me there to scan them."`
```

				goto "final choice"

			label "kaatrij kijran"
			`	Saajret laughs, a quiet ruk-ruk sound that doesn't seem very mirthful. "'Twas the Kaatrij of old that engineered them, was it not? Is it not sensible that they would be important to the House, worthy of a cast gaze even in a fraught time?" They pause for a moment, a swirl of indeterminate colors on their face. "What say'st thou to the House's offer?" they finally say.`
				goto "final choice"

			label "final choice"
			choice
				`	"All right, I'll do it."`
				`	"I'm still not willing to go back."`
					decline

			`	"Kaatrij will be pleased," says Saajret. "I shall ready myself to leave and meet thee then at thy craft."`
				accept

	on complete
		conversation
			`	You land your ship on the outskirts of the same facility you visited with Sieasej in the past; it looks much the same, though now many of the green plants have turned various shades of yellow and orange as the seasons have changed. Saajret meets you in your ship's cockpit, now dressed in a slightly different black, well-pocketed outfit. They carry a device with a rotating drum in one of their arms, probably some sort of scanner.`

> 			`	"I shall keep in contact with thy craft," they say, "though I expect no problems in my expedition. Expect me back about three hours hence."`

Reviewer 3: ```suggestion
			`	"I shall keep in contact with thy craft," they say, "though I forsee no problems in my expedition. Expect me back about three hours hence."`
```

			`	It's a remarkably peaceful vista and only becomes more so as the day wears into late afternoon and the autumn color is amplified by the golden sunlight. In the distance you see what you recognize as a Kijra-riita flitting between trees; it laps nectar from a flower on a shrub and playfully bats at a twig full of dried leaves, knocking a few to the ground. Perhaps it's the same one you released the last time you visited.`
				goto saajret

			label inside
			branch "killed riiria"
				or
					has "Successors: Ghosts Myurej 4: done"
					has "Successors: Ghosts Myurej 4: active"

			`	You check over the <ship>'s various internal systems, making sure its power couplings are operating within normal parameters and taking a long moment to admire its jump drive, which hums away inside its housing, inscrutable as always.`
				goto saajret

			label "killed riiria"
			action
				"successors: ghosts suspicion" ++
			`	You check over the <ship>'s various internal systems, and discover a mysterious anomaly in one of the power consumption logs, though it's hard to track down what caused it or where it was located. Regardless, the anomaly resolved itself some time ago.`

			label saajret

> 			`	Saajret soon returns as they promised, apparently unharmed. They unload their equipment in the quarters you set aside for them before returning to you in the cockpit. "It went without incident," they say, a shade of satisfaction on their skin. As they turn to go away, they catch themself, turning back around with a bit of trepidation. "I may never get this chance again, so I'll speak now: I've a personal request to beg of thee. There's another world within the Shroud where in past my ancestors resided. 'Fore we return to Kaatrij, I'd long to see it for myself. I'd pay thee for the diversion, of course."`

Reviewer: ```suggestion
			`	Saajret soon returns as they promised, apparently unharmed. They unload their equipment in the quarters you set aside for them before returning to you in the cockpit. "It went without incident," they say, a shade of satisfaction on their skin. As they turn to go away, they catch themself, turning back around with a bit of trepidation. "I may never get this chance again, so I'll speak now: I've a personal request to beg of thee. There's another world within the Shroud where in the past my ancestors resided. 'Fore we return to Kaatrij, I'd long to see it for myself. I'd pay thee for the diversion, of course."`
```

			choice
				`	"Sure, I'll take you."`
					goto helped
				`	"Sorry, you'll have to figure something else out."`
			`	Saajret struggles to keep their disappointment from showing. "Very well," they manage to say before leaving.`
				goto end

			label helped
			action
				set "Successors: Ghosts Helped Saajret"
			`	Saajret keeps their composure for a second before a faint pink color creeps up their face and arms. "My thanks to thee, Captain," they say. "The world is Osolaa-Val-Mavra; I'll give thee its place."`

			label end
			`	With nothing left to do here, you prepare to leave.`

mission "Successors: Ghosts 7a"
	entering
	clearance
	name "Detour to Osolaa-Val-Mavra"
	description "Saajret has asked to visit <stopovers> as a personal favor, before returning home to <planet>."
	stopover "Osolaa-Val-Mavra"
	destination "Raaqa-Puan-Uuoru"
	passengers 1
	to offer
		has "Successors: Ghosts 6: done"
		has "Successors: Ghosts Helped Saajret"
	on stopover

> 		log "Took Saajret to the Predecessor planet Osolaa-Val-Mavra. They visited an abandoned city on the surface where an ancestor of theirs once lived, retrieving from it a data-sphere and small wooden box."

Reviewer: ```suggestion
		log "Took Saajret to the Predecessor planet Osolaa-Val-Mavra. They visited an abandoned city on the surface where an ancestor of theirs once lived, retrieving from it a data-sphere and a small wooden box."
```

		conversation
			`You set the <ship> on one of the landing terraces of the planet's equatorial trench, near a location that Saajret indicates. The spot provides a dizzying view: snowflakes from a cloud overhead drift over ledges and deserted balconies before falling kilometers down into the abyss below. Saajret joins you by the airlock, now wrapped warmly in a shiny silver outfit with a collar of small ribbons that stream in the breeze.`
			`	"What is this place?" you ask.`

> 			`	A blue tone makes its way over Saajret's face. "'Twas an arc of homes and shops before they were abandoned," they say. A couple of snowflakes land on their skin and their arms wriggle against the white ground. "Thou may'st join me if thou desirith. Kaatrij demands no secrecy from us here."`

Reviewer: ```suggestion
			`	A blue tone makes its way over Saajret's face. "'Twas an arc of homes and shops before they were abandoned," they say. A couple of snowflakes land on their skin and their arms wriggle against the white ground. "Thou may'st join me if thou desirest. Kaatrij demands no secrecy from us here."`
```

			choice
				`	(Go with them.)`
				`	(Stay.)`
					goto stay
			`	The two of you make your way through the abandoned streets. You are struck by how well-preserved everything is; more than those of any other Predecessor world, the buildings here seem only temporarily vacant, as if the two of you had merely caught the city on a day with everyone missing. A trio of little cleaning robots run around the rim of a fountain, polishing away the snow before it even has time to melt in the sun.`

> 			`	Saajret follows their own path, periodically checking what must be a map on a datapad they carry. Occasionally their skin will erupt in color as they see something that interests them, but they never stay distracted for long. Eventually the two of you arrive in an elegant square of organic buildings; Saajret checks their map a few times before declaring, "here we are."`

Reviewer: ```suggestion
			`	Saajret follows their own path, periodically checking what must be a map on a datapad they carry. Occasionally their skin will erupt in color as they see something that interests them, but they never stay distracted for long. Eventually the two of you arrive in an elegant square of organic buildings; Saajret checks their map a few times before declaring, "Here we are."`
```

			choice

> 				`	"And where is 'here?'"`

Reviewer: ```suggestion
				`	"And where is 'here'?"`
```

				`	"It's a pretty spot."`
				`	(Stay silent.)`
					goto building

			`	"An ancestor of mine lived here briefly," they say. "One of import in the last days of House Sejra. But when they lived here, all that was still to come."`

			label building
			`	Saajret walks inside the largest building, entering a code onto a spherical terminal beside the door. You move to go after them, but they hold out an arm to stop you. "This last part, I would prefer to go alone," they say. They point to a high bench outside.`
			`	They stay inside for about half an hour; you watch the snow fall on the square and the terraces. When Saajret returns, they are carrying a data-sphere and a small wooden box that they carry cradled in one of their arms. There's a color on their face you can't quite place.`
			choice
				`	"Are you ready to return?"`
				`	"What did you get?"`
					goto nosy

> 			`	They hum. "Yes," they say. "I have what I came for. Thanks to thine indulgence of me."`

Reviewer: ```suggestion
			`	They hum. "Yes," they say. "I have what I came for. Thank you for thine indulgence of me."`
```

			choice
				`	"Why did you allow them to develop something so destructive?"`
					goto allow
				`	"How did you keep yourselves hidden for so long?"`
					goto hiding

			label allow
			`	"They were then upon the precipice of a great change. Their long-wrought divisions were melding; they had turned away from the gruesome vagaries of war, and were moved to explore and preserve the arts of others. We hoped," and the Quarg imparts a heightened tone on this word, "that in time, they could become as we are, guardians and friends to all. Our judgment was clouded by our desire for this."`
			choice
				`	"It's hard to believe they were ever so unified."`
				`	"You saw them as equals to yourselves?"`
			`	A demeanor of great reflection passes over the Quarg, and then is gone just as quickly. "The Myiara were like... children, just beginning to conceive of and adopt the responsibilities that lie beyond youth. Full of cleverness and arrogance still, but borne aloft by a new-blooming capacity to see beyond themselves and their own base concerns."`
			choice
				`	"How did you keep yourself hidden from them for so long?"`

			label hiding

> 			`	"Hiding indeed does not come naturally to us," the Quarg says. "With no other have we had need to conceal ourselves. One of their houses, when it knew what they had done, agreed to work with us to preserve their species. They were most skilled in matters of illusion, and shared their cloaks with us to obscure our presence."`

Reviewer 3: ```suggestion
			`	"Hiding indeed does not come naturally to us," the Quarg says. "With no other have we had need to conceal ourselves. One of their Houses, when it knew what they had done, agreed to work with us to preserve their species. They were most skilled in matters of illusion, and shared their cloaks with us to obscure our presence."`
```

	destination "Qasa-Sija-Iri"
	to offer
		has "Successors: Ghosts 9: declined"
		has "successors: ghosts dealt with aqrabe"
	on offer
		conversation
			`As disembark from your ship, you notice a group of Successor soldiers emerging from another craft in the spaceport, a Seiitej bearing House Kaatrij's colors. They meet you outside your own vessel, their weapons held at attention.`
			`	"Captain <first> <last>," their leader says, "We've been instructed to take thee with us."`
			choice
				`	"For what purpose exactly?"`
					goto purpose
				`	"And if I refuse?"`
			`	"That would not be wise," the officer says. "Thou shalt understand, we trust?"`
				goto interrogation

			label purpose
			`	"Kaatrij wishes to speak with thee," says the lead officer. "And to hear what thou hast to say."`

			label interrogation
			`	With apparently no choice, you accompany the officers back to their ship; they remove your sidearm and take you to a room inside. Two stand guard over you while the rest attend to other things outside, and it's only after the better part of an hour that someone else finally enters, a tall Successor who looks over you with an incisive glare.`

> 			`	"Captain <first> <last>," they say. "How did'st thou acquire cloaking technology?"`

Reviewer: ```suggestion
			`	"Captain <first> <last>," they say. "How didst thou acquire cloaking technology?"`

			choice
				`	"It comes from another society, far away from here."`
				`	"I was given it by a human government; they found it in an alien cargo hold."`
					to display
						has "outfit (all): Cloaking Device"
			`	"So it was not given to thee by a faction here, among the Successors?"`

> 			`	"No," you say. The Successor relaxes, if only a little. "We will search thy ship t'confirm this," they say. "But I do not believe thou lieth."`

Reviewer: ```suggestion
			`	"No," you say. The Successor relaxes, if only a little. "We will search thy ship t'confirm this," they say. "But I do not believe thou liest."`
```

			branch "cloaks"
				has "flagship attribute: cloak"

			choice
				`	"You won't find anything, my current ship isn't cloak-capable."`
			`	The Successor looks over you with a curious gaze. "Intriguing," they say. "'Tis no matter; we shall consult our records of thy craft."`

			label cloaks

> 			`	After a moment passes, your interrogator leans over you and the guards in the back stand, attentive. "Now we must decide what is to be done with thee," they say, "now that thou hast learned of our arrangement with the Quarg. Thou should'st understand we do not wish that information to be avail'ble to all. Not all the Houses nor their members regard the Quarg with amity."`

Reviewer: ```suggestion
			`	After a moment passes, your interrogator leans over you and the guards in the back stand, attentive. "Now we must decide what is to be done with thee," they say, "now that thou hast learned of our arrangement with the Quarg. Thou shouldst understand we do not wish that information to be avail'ble to all. Not all the Houses nor their members regard the Quarg with amity."`
```

			choice
				`	"I have no intention of spilling your secret."`
					goto discreet
				`	"I'll keep my lips sealed, for a price."`
					goto blackmail
				`	"The people deserve to know about this."`

> 			`	"Perhaps they shall one day, but neither thee nor I should decide that now. 'Tis a time far too fraught to bring such a thing to light. No," they say, one of their arms moving toward you, "but Kaatrij would be willing to offer thee an alternative: a formal relation with the House, that thou might be brought into our circle of knowledge and receive our boons. And, of course, be subject to our directives, including those of silence."`

Reviewer: ```suggestion
			`	"Perhaps they shall one day, but neither thee nor I should decide that now. 'Tis a time far too fraught to bring such a thing to light. No," they say, one of their arms moving toward you, "but Kaatrij would be willing to offer thee an alternative: a formal relation with the House, that thou mightst be brought into our circle of knowledge and receive our boons. And, of course, be subject to our directives, including those of silence."`
```

			choice
				`	"And what would House Sioeora think of this?"`
					to display
						has "successors sioeora patron"
					goto sioeora
				`	"What would that entail, exactly?"`
					goto details
				`	"Yes, if it finally gets me some answers."`
					goto yes
				`	"No, I'm not willing to lie for you."`
					goto no

			label blackmail

> 			`	"Thou should'st be careful with such propositions, Captain," they say. Their skin remains a murky, ashen blue. "Kaatrij would be willing to offer thee an alternative, however: a formal relation with the House, that thou might be brought into our circle of knowledge and receive our boons. Of course, thou woulds't also be subject to our directives, including those of silence. Would'st that be sufficient reward for thine efforts?"`

Reviewer: ```suggestion
			`	"Thou shouldst be careful with such propositions, Captain," they say. Their skin remains a murky, ashen blue. "Kaatrij would be willing to offer thee an alternative, however: a formal relation with the House, that thou mightst be brought into our circle of knowledge and receive our boons. Of course, thou wouldst also be subject to our directives, including those of silence. Wouldst that be sufficient reward for thine efforts?"`
```

			choice
				`	"And what would House Sioeora think of this?"`
					to display
						has "successors sioeora patron"
					goto sioeora
				`	"And what would that entail, exactly?"`
					goto details
				`	"Yes, if it finally gets me some answers."`
					goto yes
				`	"No, I think I'll stick to credits."`
					goto no

			label discreet
			action
				set "Successors: Ghosts: discreet"

> 			`	"Most discreet, Captain," they say. "But forgive me for not taking thy word alone. To that, Kaatrij is willing to make thee an offer: a formal relation with our House, that thou might be brought into our circle of knoweldge and receive our boons. And in turn, thou would'st also be subject to our directives, including those of silence. An arrangement, Kaatrij believes, of mutual benefit."`

Reviewer: ```suggestion
			`	"Most discreet, Captain," they say. "But forgive me for not taking thy word alone. To that, Kaatrij is willing to make thee an offer: a formal relation with our House, that thou mightst be brought into our circle of knoweldge and receive our boons. And in turn, thou wouldst also be subject to our directives, including those of silence. An arrangement, Kaatrij believes, of mutual benefit."`
```

			choice
				`	"And what would House Sioeora think of this?"`
					to display
						has "successors sioeora patron"
					goto sioeora
				`	"And what would that entail, exactly?"`
					goto details
				`	"Yes, if it finally gets me some answers."`
					goto yes
				`	"No, I prefer to remain a free agent."`
					goto no

			label sioeora
			`	Incendiary yellow speckles dot the Successor's face, and they continue with some small hint of satisfaction. "No doubt they would be displeas'd to have lost one they so dearly wish'd to possess. But they have fail'd to act with swiftness. What sayest thou to our offer?"`
			choice
				`	"All right, I'll do it."`
					goto yes
				`	"No, I don't want to get any more tangled up in this than I already am."`
					goto no

			label details

> 			`	"Thou would'st be counted as an alien vassal of House Kaatrij, thus granted access to our facilities, subject to our rules and directives, and empowered to fight our enemies on our behalf under a commission of war. Thou would retain thy freedom of movement, thine ability to go elsewhere in the galaxy. But when thou dost travel 'tween the stars of the Successors, 'tis House Kaatrij that thou shalt call thy home."`

Reviewer: ```suggestion
			`	"Thou wouldst be counted as an alien vassal of House Kaatrij, thus granted access to our facilities, subject to our rules and directives, and empowered to fight our enemies on our behalf under a commission of war. Thou wouldst retain thy freedom of movement, thine ability to go elsewhere in the galaxy. But when thou dost travel 'tween the stars of the Successors, 'tis House Kaatrij that thou shalt call thy home."`
```

			choice
				`	"All right, I accept."`
					goto yes
				`	"No, I don't want to get any more tangled up in this than I already am."`
					goto no

			label yes
			action
				set "successors kaatrij vassal"
				event "successors: ghosts unrestrict"
				"reputation: House Kaatrij" >?= 2
			`	"Excellent," the Successor says. "I would have thee come with us, then, to the moon where this all began, our base on <planet>. There, some of thy no-doubt-pressing questions shall be answered."`
				accept

			label no
			branch "was discreet"
				has "Successors: Ghosts: discreet"

> 			`	"A shame," your interrogator says, making a motion with one of their arms; behind them, one of the guards raises their rifle. Instinctually, you go to grab your own sidearm, only to remember that it was taken from you minutes ago just as the guard's railgun round pierces your body.`

Reviewer: ```suggestion
			`	"A shame," your interrogator says, making a motion with one of their arms; behind them, one of the guards raises their rifle. Instinctively, you go to grab your own sidearm, only to remember that it was taken from you minutes ago just as the guard's railgun round pierces your body.`⁹
```

				die

			label "was discreet"
			`	Your interrogator regards you thoughtfully for a moment. "Consider thy choice carefully, Captain," they say. "Among us, there is little way forward for a 'free agent,' as thou namest it. An offer such as this one shall not come again."`
			choice
				`	"All right, then, if it's the only way forward for me."`
					goto yes
				`	"No, I've made my decision. I won't join you."`

			action
				set "successors free agent" # This doesn't do anything now, but if Ghosts is ever expanded to let the player character ally with other Houses, this tag would be useful to have.

> 			`	"A shame," the Successor says. "But so long as thou does not give up what thou hast learned or otherwise make thyself an enemy of this House, Kaatrij shall not impede thee." They pause and their skin blossoms with intense color. "Do not make us regret this leniency."`

Reviewer: ```suggestion
			`	"A shame," the Successor says. "But so long as thou dost not give up what thou hast learned or otherwise make thyself an enemy of this House, Kaatrij shall not impede thee." They pause and their skin blossoms with intense color. "Do not make us regret this leniency."`
```

			`	They produce a data-sphere which they give over to you. It feels slightly cold to the touch. "Thine acceptance binds thee to this House as vassal; herefore it shall expect of thee thine support and secrecy, and thou might expect from it the powers and privileges thou might require. Dost thou accept?"`
			choice
				`	"I do."`
					goto yes
				`	"Maybe this was a bad idea."`
					goto uncertain
				`	"Does this mean you'll sell me weapons and warships?"`

			`	A color flits upon Kampa's face, but you can't quite tell if it's amusement or suspicion. "Strictly, yes, but such exchanges are governed by treaty. For the moment, while thine alignment with us is still unclear to the rest, Kaatrij declines this responsibility."`
			choice
				`	"All right, I formally accept."`
					goto yes
				`	"Maybe this was a bad idea."`

			label uncertain
			`	"Thou hast already seen this place, Captain, hast already accepted in word if not in bond. Thou art... too far in already."`
			choice
				`	"I suppose I am."`

			label yes

> 			`	The data-sphere flares with color, registering your assent. "'Tis done then," Kampa says. "Welcome to House Kaatrij." Both Kampa and Saajret bow again, the Quarg the only one in the room remaining motionless. "Thou shalt have access to much of our works, which now thou might'st explore at thy leisure. When thou'rt ready, thou should'st meet Saajret at the spaceport here for thy first official task for us."`

Reviewer: ```suggestion
			`	The data-sphere flares with color, registering your assent. "'Tis done then," Kampa says. "Welcome to House Kaatrij." Both Kampa and Saajret bow again, the Quarg the only one in the room remaining motionless. "Thou shalt have access to much of our works, which now thou mightst explore at thy leisure. When thou'rt ready, thou shouldst meet Saajret at the spaceport here for thy first official task for us."`
```

					to display
						not "Successors: drone"
					goto drone
				`	"What exactly is this arrangement with the Quarg that you have?"`
					to display
						not "Successors: quarg"
					goto quarg
				`	"How much cloaking technology do you actually have?"`
					to display
						not "Successors: cloak"
						has "Successors: attack"
					goto cloak
				`	"Did you attack House Seineq?"`
					to display
						has "Successors: attack"
						has "Successors: drone"
						has "Successors: cloak"
						not "Successors: seineq"
					goto seineq

			label attack
			action
				set "Successors: attack"
			`	"'Twas one of the High Houses, but which exactly, we do not know yet for certain." Kampa contemplates for a moment. "Each of the Houses keeps a few heirloom warships from the time of the old empire, and most dare not bring them from the gilded vaults where they lie, for they are far too precious to risk in petty conflicts. Yet another still fielded one of theirs, raided the moon, killed hundreds, all to steal a single cloaking unit. Now they flit about unseen."`
				goto questions

			label seineq
			action
				set "Successors: seineq"

> 			`	There's a pause as Kampa looks at you with incisive eyes. "Thou'rt fair to ask it," they say. "No," they say, "'twas not Kaatrij that struck them. Once our cloak was stolen, our 'lusive foe had all 'twas needed t'frame us of that trait'rous blow: t'let themselves been seen with signs that we alone had once posses'd."`

Reviewer: ```suggestion
			`	There's a pause as Kampa looks at you with incisive eyes. "Thou'rt fair to ask it," they say. "No," they say, "'twas not Kaatrij that struck them. Once our cloak was stolen, our 'lusive foe had all 'twas needed t'frame us of that trait'rous blow: t'let themselves be seen with signs that we alone had once posses'd."`
```

			choice
				`	"Do you have any data on the drone ships?"`
				`	"We'd better return, then."`
					goto end

			`	"A good thought, captain," Saajret says. A difficult-to-identify emotion flits over their skin as they return to the terminal. They download some of the biographical data they were consulting into a portable data-sphere and, once they have tracked down data pertaining to the ships themselves, download that as well.`
			`	The drone ships themselves were designed by a Myiara named Resja, a noted shipwright for the old House Aqrabe. Initially the craft were designed for small crews but after Resja's death - apparently by poisoning - they were retrofitted with drone cores and robotic maintenance systems, after which they served first as expendable rearguards and finally as the lingering defense systems that you've encountered.`
			`	The biographical data is added to your log.`

			action
				"Successors: Ghosts Saajret Forgiveness" -= 2
				set "successors: ghosts resja biography"

> 				log "People" "Resja" "Resja was a noted Myiaran engineer and shipwright who worked soon before the collapse of the empire of the Myiaran Predecessors circa 1600. They are responsible for designing the originally-crewed drone ships that linger in Predecessor space, later retrofitted as self-guided craft after Resja's death by poisoning."

Reviewer 3: ```suggestion
				log "People" "Resja" "Resja was a noted Myiaran engineer and shipwright who worked soon before the collapse of the empire of the Myiaran Predecessors, circa 1600. They are responsible for designing the originally-crewed drone ships that linger in Predecessor space, later retrofitted as self-guided craft after Resja's death by poisoning."
```

					to display
						not "Successors: security"
					goto security
				`	"How could it have been revealed in the first place?"`
					to display
						not "Successors: revealed"
					goto revealed
				`	"What's the second task?"`
					goto second

			label security
			action
				set "Successors: security"
			`	Saajret prods a flower on the floor with the tip of one of the arms. "I've been fashioned with keys to elude the station's own defenses. That which poses the most threat to us are the independent ones; thou hast before met the guardian drone ships? With the citadel reveal'd, no doubt one has already taken up residence there."`
				goto "questions 1"

			label revealed
			action
				set "Successors: revealed"
			`	"That may be the most pertinent question that faces us all now, Captain." Many-colored waves ripple over Saajret's skin. "The location of our citadel, I am now told, was known to precious few within Kaatrij and none without. It suggests what I've suspected all along; one within Kaatrij has betrayed the House, sold our secrets to another."`
				goto "questions 1"

			label second
			action
				clear "Successors: security"
				clear "Successors: revealed"

> 			`	"Secondly," says Saajret, "'tis clear we must end this flitting about in shadow of our foe. While thou and I breach the citadel, Kaatrij and the Quarg here plan a trap to snare an heirloom warship. To destroy such a craft would be a great wound to our attacker, and would, if we are fortunate, give us some measure of their identity.`

Reviewer 3: ```suggestion
			`	"Secondly," says Saajret, "'tis clear we must end this flitting about in the shadow of our foe. While thou and I breach the citadel, Kaatrij and the Quarg here plan a trap to snare an heirloom warship. To destroy such a craft would be a great wound to our attacker, and would, if we are fortunate, give us some measure of their identity.`
```

			`	The inside is truly cavernous; the circular vault of the station's core forms an artificial sky, the landing pad for your ship a stamp-sized rectangle on its perimeter. Saajret exits from their landed ship and meets you with a pair of Kaatrij soldiers; you recognize one as having accompanied Kampa before when they interrogated you.`
			`	"I've added us as allies in the station's security rolls," says Saajret. "It should permit us access below." They check a data-pad that they produce from one of their pockets, looking from the shifting metallic screen to the layout of the docking bay. Eventually, they point to one of several circular apertures on the floor. As you approach, a large elevator pod emerges from it and its doors open expectantly.`
			choice
				`	"What's with the Myiara and these massive constructions?"`
				`	"How far down are we going?"`
					goto down

> 			`	Saajret is speckled with mirth and hums. "Myiara?" they say. "'Twas the Quarg that told thee this, no doubt. Ah, but to answer thy query: for Kaatrij at least, 'twas a simple matter of geometry. To cloak a station is a function of its armor, its surface - the power to feed that cloak is a function of its volume. A larger citadel was more efficient to hide away." They rub the tips of two of their arms over a data-sphere embedded in the elevator pod.`

Reviewer 3: I think this contradicts existing lore on cloaks. Unless I'm misremembering, the Remnant state that for them it's easier to cloak smaller ships with a smaller surface area to volume ratio, the exact opposite of what Saajret is saying here. Maybe their cloaks operate on different principles, I'm not sure.

				`	(Rip off the translation device.)`
				`	(Try to block out the sounds.)`
			`	As soon as you move, something punches against your inner ear and you crumple to the ground. The screaming voice stops, and another one takes its place, less pointed but no less painful. "We recognize thy kind," it says. "And the other with thee, too. Saajret?" As their name is spoken, Saajret stands over you, their words unintelligible. The voice shifts again. "Deceiver!" it screams. "Thou who hast plundered beast and bone from where they should'st forever lie!"`
			choice
				`	"I'm trying to stop the thieves!"`
				`	"How are you in my head!?"`
				`	"What about Saajret?"`
			`	The rage of the artificial voice chills. "Insignificant," it shouts, and as something else begins to interleave with the sound, something sharp and painful, there comes a quiet final phrase: "I missed thee," the voice says. "I missed thee."`
			`	And all of a sudden it stops; the pain, the voice. The pressure fades away from your head and Saajret looks down at you, the translation device plucked from your body and held in their arm. They help you to your feet and return it to you; you store it in a pocket, deactivated.`

> 			`	Saajret confers with their guards for a moment, their words a mess of unintelligible sound. One of them interacts with the elevator controls and the carriage starts to move upwards again, the station's com system shrieking to no avail. Saajret comes to speak to you, gently poking your chest with the tip of one of their arms; they say something that sounds a bit like your name. Suddenly the elevator is shaken by an impact from outside and soon after, dozens more. The four of you are deposited back in the landing bay just as you catch a glimpse of a ship in the distance firing on the station. You dash back to your own craft and prepare to flee or fight.`

Reviewer 3: ```suggestion
			`	Saajret confers with their guards for a moment, their words a mess of unintelligible sound. One of them interacts with the elevator controls and the carriage starts to move upwards again, the station's com system shrieking to no avail. Saajret comes to speak to you, gently poking your chest with the tip of one of their arms; they say something that sounds a bit like your name. Suddenly the elevator is shaken by an impact from outside, and soon after dozens more follow. The four of you are deposited back in the landing bay just as you catch a glimpse of a ship in the distance firing on the station. You dash back to your own craft and prepare to flee or fight.`
```

			`	Kampa considers this for a moment. "The benefit of thou joining us outweighs the loss of a small delay," they say. "We can afford to wait on thee to acquire such a craft if it enables this ambush. Please act quickly if thou wish t'assist."`
				defer

			label deferred
			branch "still no cloak"
				not "flagship attribute: cloak"

			`	Having acquired a cloak-capable flagship, you are quickly directed back to a briefing room within the base where Kampa reiterates the plan: you will join a fleet of House Kaatrij's ships to deploy Quarg cloak detectors around the wormhole, before laying in wait on <destination> to ambush the heirloom warship that attacked Kaatrij.`
				accept

			label "still no cloak"
			`	Without a cloak-capable flagship, you cannot join House Kaatrij in their ambush.`
				defer

	on enter "Oublaa-Khora"
		dialog `As you exit the wormhole, you trigger the release of the cloak detector from your ship's cargo bay. The metal orb floats into position at the wormhole's edge before disappearing from view entirely as its cloaking armor activates. Apparently satisfied, Saajret directs you to land on the nearby planet.`

	on complete
		conversation

> 			`Once the rest of House Kaatrij's fleet has deployed their own cloak detectors, the fleet of Vujleten follow you into <planet>'s atmosphere, bedding down in the shadow of an ancient weather control tower to shield them from the dust. Several of the Successors inside exit to inspect the tower's functioning and after a few minutes return. They bring long fuel hoses out of their ships and connect them to ports on the tower's side; one by one, the flock of Vujleten shimmer and then disappear as their own cloaks engage.`

Reviewer 3: ```suggestion
			`Once the rest of House Kaatrij's fleet has deployed their own cloak detectors, the fleet of Vujleten follow you into <planet>'s atmosphere, bedding down in the shadow of an ancient weather control tower to shield them from the dust. Several of the Successors inside exit to inspect the tower's functioning, and return after a few minutes. They bring long fuel hoses out of their ships and connect them to ports on the tower's side; one by one, the flock of Vujleten shimmer and then disappear as their own cloaks engage.`
```

			`	Kampa speaks first as they lead you away from your ship and into a more secluded region of the port. "'Tis good that thou prevail'd," they say, sharing in Saajret's satisfaction. "'Twas a fraught battle, I trust?"`
			choice
				`	"Indeed."`
					goto next
				`	"I've fought tougher."`

			`	"Indeed?" Kampa says. "Later thou shalt have to tell us more of this."`

			label next
			`	They produce a data-sphere from their outfit and offer it to you. As you take it, they continue, "I shall enjoy debriefing thee after our conversation here, but for now, Kaatrij has seen fit to offer thee a reward for thy able service. Since thou has yet to receive the ability to purchase our technologies, today 'tis purely monetary. The sphere authorizes the transfer of <payment> to thee."`

> 			choice

Reviewer 3: Feels like there should be an opportunity to ask about whether they've come any closer to figuring out which High House was behind the attack.

				accept
	on complete
		log "Attended the wedding between scions of Houses Chydiyi and Myurej. Was presented with a license to purchase advanced Successor technology."
		set "license: High Houses"
		event "successors: ghosts intermission timer" 40
		event "successors: ghosts kaatrij expansion"
		conversation
			branch waterlining
				has "flagship attribute: waterlining"
			`You are given directions to one of the shallow-water landing pads on a secondary <planet> spaceport, from which you once again take a long elevator deep into the ocean. Outside the window dance the "Ribbons of Iret," enormous plumes of color rising up from from the abyss, only growing more intense the farther down you travel.`
				goto complex

			label waterlining
			`Traffic control gives you coordinates to the deepwater complex where the wedding is to take place, and you plunge the <ship> into the planet's ocean, its hull wrapped around by one of the "Ribbons of Iret," enormous and colorful. The farther down you travel, the more intense the bioluminescent plume grows, until it feels almost as if you are traveling through a nebula in space rather than an underwater bloom.`

> 			`	You reach the complex and dock at it, disembarking through a complex airlock and taking an elevator the final short distance.`

Reviewer 3: ```suggestion
			`	You reach the complex and dock at it, disembarking through an airlock and taking an elevator for the final short distance.`
```

			label complex

> 			`	You soon arrive at a similar complex to the one you visited the last time you were here, though this one is even larger and more richly decorated, the walls a riot of color and form that is as ostentatious as it is impressive. A gentle mist issues from apertures on the vaulted ceiling and long golden threads trail downwards, writhing around particular Successors and eventually, you.`

Reviewer 3: The first sentence sounds a bit awkward right after line 1860. How about:
```suggestion
			`	You soon enter a similar room to the one you visited the last time you were here, though this one is even larger and more richly decorated, the walls a riot of color and form that is as ostentatious as it is impressive. A gentle mist issues from apertures on the vaulted ceiling and long golden threads trail downwards, writhing around particular Successors and eventually, you.`
```

					goto appreciate
				`	"It's a very impressive facility you have here."`
			`	They hum. "Such beauty earns the envy of many; the natural and artificial intertwined."`
				goto khiajsore

			label appreciate
			`	"For such a unique and industrious figure, 'twould've been an unbearable snub not to!"`

			label khiajsore
			`	They rattle one of their arms a little, red gold and sparkling stones swinging through the air. "I just spoke of thee, in fact, with a delegate of House Seineq here, who confirmed what I've heard that thou were't of use to them as well! I think," they say, "thou set a good example for future aliens - benevolent, equitable, humble. Though I hear now Kaatrij has obtained thy vassalage? A boon to them, no doubt." Their face sparkles with color, eyes alight with furtive interest. "Though I hope for our efforts, Myurej has earned some measure of thy favor."`
			choice
				`	"I'm happy to help, so long as it doesn't conflict with my other responsibilities."`
				`	"I'm not sure I like where this is going."`
			`	Khiajsore laughs. "Of course, of course," they say. "Thou hast already helped us greatly. Now please excuse me."`

> 			`	They leave and two members of the crowd break off the follow them, which you notice carry small weapons strapped to their outfits. One of the golden threads makes its way back to you, informing you that the wedding ceremony will be beginning soon.`

Reviewer 3: ```suggestion
			`	They leave and two members of the crowd break off to follow them, which you notice carry small weapons strapped to their outfits. One of the golden threads makes its way back to you, informing you that the wedding ceremony will be beginning soon.`
```

			choice
				`	(Attend the wedding.)`
				`	(Wait around until the later meeting.)`
					goto skipped
			`	While most Successors are waiting outside the entrance to what must be a massive room, you are instead guided to an inconspicuous and out-of-the-way door, guarded by a pair of Successor soldiers. Neither one reacts to your attempt to pass, and as you step behind the door, it becomes clear why you aren't lining up with the general public: that massive room is entirely filled with water, a gigantic sphere with lights, platforms, and countless seats all built within it. A few small rooms like the one you now stand in are built into the walls like little transparent air bubbles at the shore of a great sea.`

> 			`	You are joined by three other Successors after a few minutes, each of whom takes their place in the little room. The ceremony begins shortly after. From apertures in the bottom of the sphere, legions of soldiers emerge, half wearing the colors of House Chydiyi and the other half of House Myurej. The two small armies swim around each other, interleaving their lines of bodies like the Ribbons themselves that you saw outside.`

Reviewer 3: Not sure if the capitalization here was intentional; feel free to ignore this suggestion if it was.
```suggestion
			`	You are joined by three other Successors after a few minutes, each of whom takes their place in the little room. The ceremony begins shortly after. From apertures in the bottom of the sphere, legions of soldiers emerge, half wearing the colors of House Chydiyi and the other half of House Myurej. The two small armies swim around each other, interleaving their lines of bodies like the ribbons themselves that you saw outside.`
```

> 			`	Only after a half hour of this acrobatics do the couple finally appear, rising up from the same apertures and joined by a flanking group of what must be the scions of their respective houses. You spy Khiajsore among them. There is a further intertwining, a display of gold and colors, and then a collective hum rises from the crowd.`

Reviewer 3: ```suggestion
			`	Only after a half hour of this acrobatics does the couple finally appear, rising up from the same apertures and joined by a flanking group of what must be the scions of their respective houses. You spy Khiajsore among them. There is a further intertwining, a display of gold and colors, and then a collective hum rises from the crowd.`
```

> 			`	Only after a half hour of this acrobatics do the couple finally appear, rising up from the same apertures and joined by a flanking group of what must be the scions of their respective houses. You spy Khiajsore among them. There is a further intertwining, a display of gold and colors, and then a collective hum rises from the crowd.`

Reviewer 3: What does `a display of gold and colors` mean? Is gold not a color?

				`	"It's good to see you two getting along."`
					to display
						not "Successors: getting along"
					goto "getting along"
				`	"Do you think Aqrabe and Kaatrij will be collaborating much?"`
					to display
						not "Successors: collaboration"
					goto collaboration
				`	"I'm still uncertain joining Kaatrij was the right choice."`
					to display
						not "Successors: uncertain"
					goto uncertain
				`	"I should get going."`
					goto end

			label "getting along"
			action
				"Successors: Ghosts Saajret Openness" ++
				set "Successors: getting along"
			`	Saajret nods. "Tiliia's work on the keystones that thou provided intrigues me greatly. It bears some resemblance to technologies I've studied in our archives - though the stellar metal shells are a wholly inventive solution, I believe."`
			`	Tiliia interjects at this, a hint of embarrassment on their skin. "T'was a matter of managing with what could be done," they say.`
			`	"Nonetheless," says Saajret, now to Tiliia, "A worthy achievement. I am glad for this little lull if only for the sake of collaboration."`
				goto "saajret and tiliia"

			label "collaboration"
			action
				set "Successors: collaboration"

> 			`	"'T'would be fruitful, doubtless," Tiliia says. "And now Kaatrij opens up a little for the first time since their wound, perhaps 'tis not beyond the realm of possibility."`

Reviewer 3: ```suggestion
			`	"'T'would be fruitful, doubtless," Tiliia says. "And now that Kaatrij opens up a little for the first time since their wound, perhaps 'tis not beyond the realm of possibility."`
```

			choice
				`	"Do you know what House they belonged to?"`
				`	"Is there no way to tell which one exactly was behind the attack?"`
			`	"That is where misfortune meets us," Saajret says. "All three of those were made for a combined alliance that fought in the last great inter-Houses war; when t'were disassembled and smuggled away, we know not which House of the alliance spirited each away. Much remains unclear."`
			choice
				`	"Do you have any data on the drone ships?"`
				`	"We'd better return, then."`
					goto end

			`	"A good thought, captain," Saajret says. A difficult-to-identify emotion flits over their skin as they return to the terminal. They download some of the biographical data they were consulting into a portable data-sphere and, once they have tracked down data pertaining to the ships themselves, download that as well.`

> 			`	The drone ships themselves were designed by a Myiara named Resja, a noted shipwright for the old House Aqrabe. Initially the craft were designed for small crews but after Resja's death - apparently by poisoning - they were retrofitted with drone cores and robotic maintenance systems, after which they served first as expendable rearguards and finally as the lingering defense systems that you've encountered.`

Reviewer: ```suggestion
			`	The drone ships themselves were designed by a Myiara named Resja, a noted shipwright for the old House Aqrabe. Initially, the craft were designed for small crews, but after Resja's death - apparently by poisoning - they were retrofitted with drone cores and robotic maintenance systems, after which they served first as expendable rearguards and finally as the lingering defense systems that you've encountered.`
```

	waypoint "Eta Carinae"
	waypoint "Avasaa-Novaa"
	to offer
		has "Successors: Ghosts 13: done"
	on offer
		conversation
			`Saajret meets you in another one of the spaceport's overlooking rooms, a liquid-metal starchart laid out on a table before them. It looks like it's showing areas of Predecessor space. They greet you with a splay of the arms as you enter.`
			`	"Is this my next mission?" you ask, gesturing to the starchart.`
			`	Saajret hums thoughtfully. "Yes," they say. "One of the things thou did earli'r was the placement of scanning eyes throughout the Shroud, at Seineq's dear request."`
			choice
				`	"I recall."`
					goto direct
				`	"Could Seineq be the House behind the attack?"`

			`	Whorls of pink mirth erupt over Saajret's skin. "Perhaps so, perhaps not," they say. "For now, Kaatrij's aim is more direct. The eyes make notes of jumps of the proper sort, as are made by thine own craft and the Quarg - but the flitting of our foe relies on aught uncanni'r that we believe evades their gaze. Thus, as Kampa's wont to say, they are to us a tactical disadvantage. Kaatrij seeks that thou, and I, observing, destroy them."`
				goto questions

			label direct

> 			`	"Kaatrij now seeks that thou undo'th this, with my presence as observer. 'Thout Seineq's control of those, our foe might come to watch them - 'tis a risk Kaatrij is loath to take. For they make note of jumps of the proper sort, as made by thy craft and the Quarg's alike. But the flitting of our foe relies on aught uncanni'r that we think evades their gaze. Thus, they reveal not the comings and goings of the guilty but thee and perhaps our own craft."`

Reviewer: ```suggestion
			`	"Kaatrij now seeks that thou undoeth this, with my presence as observer. 'Thout Seineq's control of those, our foe might come to watch them - 'tis a risk Kaatrij is loath to take. For they make note of jumps of the proper sort, as made by thy craft and the Quarg's alike. But the flitting of our foe relies on aught uncanni'r that we think evades their gaze. Thus, they reveal not the comings and goings of the guilty but thee and perhaps our own craft."`
```

			`	"My thanks to thee for thy help so far," Saajret says. "I hope thou hast found our little shared space here comfortable enough."`
			choice
				`	"I have."`
				`	"What's the situation with that station?"`
			`	Saajret nods. "I've conferred with Kaatrij and our Quarg patrons and, for once, we are all agreed. Two tasks lie before us, with thou included if thou wilt: first, we must assess how our citadel has been plunder'd. 'Tis well-defended, and seeing as its cloak was not fully disabled, its security must remain at least partially op'rative. For this, thou might'st play a vital role, again with thy unhind'red craft."`
			label "questions 1"
			choice
				`	"'Security?' Not sure I like the sound of that."`
					to display
						not "Successors: security"
					goto security
				`	"How could it have been revealed in the first place?"`
					to display
						not "Successors: revealed"
					goto revealed
				`	"What's the second task?"`
					goto second

			label security
			action
				set "Successors: security"

> 			`	Saajret prods a flower on the floor with the tip of one of the arms. "I've been fashioned with keys to elude the station's own defenses. That which poses the most threat to us are the independent ones; thou hast before met the guardian drone ships? With the citadel reveal'd, no doubt one has already taken up residence there."`

Reviewer: ```suggestion
			`	Saajret prods a flower on the floor with the tip of one of their arms. "I've been fashioned with keys to elude the station's own defenses. That which poses the most threat to us are the independent ones; thou hast before met the guardian drone ships? With the citadel reveal'd, no doubt one has already taken up residence there."`
```

				goto "questions 1"

			label second
			action
				clear "Successors: security"
				clear "Successors: revealed"
			`	"Secondly," says Saajret, "'tis clear we must end this flitting about in the shadow of our foe. While thou and I breach the citadel, Kaatrij and the Quarg here plan a trap to snare an heirloom warship. To destroy such a craft would be a great wound to our attacker, and would, if we are fortunate, give us some measure of their identity.`
			choice
				`	"How do you plan to catch it if it's cloaked?"`
				`	"All right, what's next?"`
					goto next
			`	The mess of colors dotting Saajret's face resolves into a unified shade of satisfaction. "The Quarg, it seems, have certain ways to uncover such hidden craft. An old foe of theirs used a similar tactic, I'm told. Convincing them to share aught is a lifetime task for the most persuasive of diplomats, but in this instance they may help us for the sake of quieting that ship's space-ripping drive. For that," they say, "they are most eager."`

			label next

> 			`	A fine mist is emitted by ports on the ceiling, watering the plants and gathering in tiny spheres on your clothes. "We should prepare for our journey to the citadel. Equip thy craft as thou desirith, then let us depart."`

Reviewer: ```suggestion
			`	A fine mist is emitted by ports on the ceiling, watering the plants and gathering in tiny spheres on your clothes. "We should prepare for our journey to the citadel. Equip thy craft as thou desireth, then let us depart."`
```

			`	Saajret mutters darkly and once again orders the soldiers to set up the scanners, and once again they project their beams, now through the transparent door and into the breached segment. A wave of black ruffles over Saajret's skin. "All but one of the station's cloaking nodes have been remov'd. So strange..." they say, "to have stolen these and yet left the vault unassailed." The wave passes, leaving only colorlessness in its wake. "We must bring this news with haste. Let us hurry back."`
			`	All of a sudden, the station's comm system crackles to life. A rapid series of interweaving clicks emerge, and they soon shift and warp in your ears as your translation device begins to interpret them. "Interloper!" something screams, and you feel dizzy. "Despoiler! Invading wretch!"`
			choice
				`	(Rip off the translation device.)`
				`	(Try to block out the sounds.)`

> 			`	As soon as you move, something punches against your inner ear and you crumple to the ground. The screaming voice stops, and another one takes its place, less pointed but no less painful. "We recognize thy kind," it says. "And the other with thee, too. Saajret?" As their name is spoken, Saajret stands over you, their words unintelligible. The voice shifts again. "Deceiver!" it screams. "Thou who hast plundered beast and bone from where they should'st forever lie!"`

Reviewer: ```suggestion
			`	As soon as you move, something punches against your inner ear and you crumple to the ground. The screaming voice stops, and another one takes its place, less pointed but no less painful. "We recognize thy kind," it says. "And the other with thee, too. Saajret?" As their name is spoken, Saajret stands over you, their words unintelligible. The voice shifts again. "Deceiver!" it screams. "Thou who hast plundered beast and bone from where they shouldst forever lie!"`
```

				launch

	on fail
		dialog phrase "successor ghosts story failed"

mission "Successors: Ghosts 17"
	landing
	name "Flee to Successor space"
	description "Escape the drone ship and return to <destination>."
	source "Ijra-Ea, Shimmering Veil"
	destination "Qasa-Sija-Iri"
	to offer
		has "Successors: Ghosts 16: done"
	npc evade
		ship "Resja, of the Nine Eyes" "Resja, Fury 16-11"
		government "Solitude Enforcer"
		personality heroic nemesis unconstrained vindictive launching
		on kill
			set "Successors: Ghosts Resja transmission"
			dialog "As the drone ship explodes it transmits some data to your own vessel. It cannot be deciphered at this time."
	npc save
		government `House Kaatrij`
		personality launching coward
		ship "Vujlet-Usaqri (Jump)" "o-Sapi, Brilliant Voice"
	on fail
		dialog phrase "successor ghosts story failed"

mission "Successors: Ghosts Scout Escorts"
	invisible
	entering
	source
		system "Oublaa-Khora"
	destination "Qasa-Sija-Iri"
	to offer
		has "Successors: Ghosts 17: active"
	npc save
		government "House Kaatrij"
		personality waiting heroic
		ship "Vujlet-Usaqri (Keystone)" "Tili, Open Arms"
		ship "Vujlet-Usaqri (Keystone)" "Luatet, Gentle Song"

> mission "Successors: Ghosts 18"

Reviewer: bookmark

			label questions
			choice
				`	"My current ship isn't able to cloak."`
					to display
						not "flagship attribute: cloak"
					goto "no cloak"
				`	"What if the fleet is disabled?"`
					to display
						not "Successors: lose"
					goto lose
				`	"What sort of fight should I expect?"`
					to display
						not "Successors: expect"
					goto expect
				`	"All right, I'm ready."`
					to display
						has "flagship attribute: cloak"
					goto end
			label lose
			action
				set "Successors: lose"
			`	Kampa nods. "We've a second plan, with crews already prepar'd. But 'tisn't thy responsibility to know it."`
				goto questions

			label expect
			action
				set "Successors: expect"
			`	Saajret is the one who speaks now. "'Tis a proper battlecruiser of our Predecessors' age," they say. "Our warships in their day were second only to those of the Quarg; one could not doubt stand against a fleet of lesser craft handily."`
				goto questions

			label end
			`	Saajret speaks. "I'll join thee, <last>, before we depart. Thou shalt need a guide to the world where we seek to plan the ambush."`
				accept

			label "no cloak"
			action
				set "Successors: Ghosts 18: deferred"

> 			`	Kampa considers this for a moment. "The benefit of thou joining us outweighs the loss of a small delay," they say. "We can afford to wait on thee to acquire such a craft if it enables this ambush. Please act quickly if thou wish t'assist."`

Reviewer: ```suggestion
			`	Kampa considers this for a moment. "The benefit of thy joining us outweighs the loss of a small delay," they say. "We can afford to wait on thee to acquire such a craft if it enables this ambush. Please act quickly if thou wish t'assist."`
```

			`	Saajret slithers off almost immediately, only offering a hasty promise of their return and a poorly-concealed tint of satisfaction. You don't have to wait long, though; by the time the spaceport's automated repair arms have folded down to assess the <ship>'s battle damage, you notice Saajret coming back, Kampa and a smattering of Kaatrij officers at their side. Though none of the base's Quarg join them, you notice one watching the scene from one of the rooms overlooking the spaceport.`
			`	Kampa speaks first as they lead you away from your ship and into a more secluded region of the port. "'Tis good that thou prevail'd," they say, sharing in Saajret's satisfaction. "'Twas a fraught battle, I trust?"`
			choice
				`	"Indeed. We were lucky it was just one ship."`
					goto next
				`	"It was. I'm sorry none of your own crews survived."`
					to display
						has "Successors: Ghosts support fleet died"
					goto mournful
				`	"I've fought tougher."`

			`	"Indeed?" Kampa says. "Later thou shalt have to tell us more of this."`

			label mournful
			`	"As are we," says Kampa. "But their names shall be honored ones for their courage and devotion."`
			label next

> 			`	They produce a data-sphere from their outfit and offer it to you. As you take it, they continue, "I shall enjoy debriefing thee after our conversation here, but for now, Kaatrij has seen fit to offer thee a reward for thy able service. Since thou has yet to receive the ability to purchase our technologies, today 'tis purely monetary. The sphere authorizes the transfer of <payment> to thee."`

Reviewer: ```suggestion
			`	They produce a data-sphere from their outfit and offer it to you. As you take it, they continue, "I shall enjoy debriefing thee after our conversation here, but for now, Kaatrij has seen fit to offer thee a reward for thy able service. Since thou hast yet to receive the ability to purchase our technologies, today 'tis purely monetary. The sphere authorizes the transfer of <payment> to thee."`
```

				goto "what now"

			label technology
			`	Saajret nods here. "That wedding 'tween Myurej and Chydiyi is likely to happen soon," they say. "Thy time of shrouded service to Kaatrij has come to its close; with all the Houses there, 'tis a fair opportunity to present thee with our license."`
			`	"Thou hast done service to other Houses, too," Kampa says, "so it should be 'thout much resistance."`
				goto "what now"

			label generous
			`	Kampa bows and Saajret interjects. "That wedding 'tween Myurej and Chydiyi is likely to happen soon," they say. "And with it, the end to thy shrouded service to Kaatrij. 'Tis a fair time to present thee with our license, as much as it may interest thee."`
			`	"Thou hast done service to other Houses, too," Kampa says, "so it should be 'thout much resistance."`

			label "what now"
			choice
				`	"What now?"`
				`	"Do you have anything else for me?"`

			branch "wedding preparations"
				not "successors: ghosts dealt with myurej"

			`	"Just to wait," Kampa says. "Kaatrij must now determine its next move."`
				goto end

			label "wedding preparations"

> 			`	"On the matter of the forthcoming wedding," Kampa says. "Myurej continues to whine about some absent gifts of theirs. Shoulds't thou need to earn thyself an invitation, that would be a matter to pursue."`

Reviewer: ```suggestion
			`	"On the matter of the forthcoming wedding," Kampa says. "Myurej continues to whine about some absent gifts of theirs. Shouldst thou need to earn thyself an invitation, that would be a matter to pursue."`
```

			label end

> 			`	With that said, Kampa bows and leaves. "I expect thee should wish to supervise thy craft's repairs," Saajret says, "or perhaps merely enjoy some rest." Their skin speckles with mirth. "Farewell, Captain. I hope to see thee again soon."`

Reviewer: ```suggestion
			`	With that said, Kampa bows and leaves. "I expect thou shouldst wish to supervise thy craft's repairs," Saajret says, "or perhaps merely enjoy some rest." Their skin speckles with mirth. "Farewell, Captain. I hope to see thee again soon."`
```

			`	The thread inspects your face and clothing for a short time before it speaks in a soft, synthesized voice. "Greetings, Captain <first> <last>! Thou'rt authorized to attend the ceremony this afternoon and, following, a special meeting of the locally-gathered Houses." The voice continues for a while, providing you with information on which rooms to visit and how you are to find your way there.`
			`	Soon after, you catch sight of Khiajsore, the Myurej scion you spoke with before. They catch sight of you as well and slither towards you, batting away one of the threads with a bejeweled arm. "Captain, captain, delightful Captain," they say. "How pleasant of thee to accept our invitation."`
			choice
				`	"I appreciated your extending it."`
					goto appreciate
				`	"It's a very impressive facility you have here."`
			`	They hum. "Such beauty earns the envy of many; the natural and artificial intertwined."`
				goto khiajsore

			label appreciate
			`	"For such a unique and industrious figure, 'twould've been an unbearable snub not to!"`

			label khiajsore

> 			`	They rattle one of their arms a little, red gold and sparkling stones swinging through the air. "I just spoke of thee, in fact, with a delegate of House Seineq here, who confirmed what I've heard that thou were't of use to them as well! I think," they say, "thou set a good example for future aliens - benevolent, equitable, humble. Though I hear now Kaatrij has obtained thy vassalage? A boon to them, no doubt." Their face sparkles with color, eyes alight with furtive interest. "Though I hope for our efforts, Myurej has earned some measure of thy favor."`

Reviewer: ```suggestion
			`	They rattle one of their arms a little, red gold and sparkling stones swinging through the air. "I just spoke of thee, in fact, with a delegate of House Seineq here, who confirmed what I've heard that thou wert of use to them as well! I think," they say, "thou didst set a good example for future aliens - benevolent, equitable, humble. Though I hear now Kaatrij has obtained thy vassalage? A boon to them, no doubt." Their face sparkles with color, eyes alight with furtive interest. "Though I hope for our efforts, Myurej has earned some measure of thy favor."`
```

			`	When they return to the edge, no more is said. Kampa returns to the pool and produces a small wooden box from their outfit, which they give to you.`
			choice
				`	(Take it.)`
			`	You open the box to find an elaborate gold brooch nestled among soft folds of velvet.`
			scene "outfit/high houses license"
			`	"Let this gift make clear to all thy responsibilities and privileges, and may thou have free access to all our House and allies offer..." Kampa says, "<first> <last>, Honored Knight-Vassal of the High House Kaatrij." A colorful display rushes over Kampa's skin and they pick up the brooch with one arm and pin it on your jacket right next to your translation device and the civilian license. The metal must be very dense; it is noticeably heavier than the others.`
			`	With this, the meeting concludes, the various members exiting in turn - the ones you've met before offering some short words of congratulations. You eventually exit as well, finding your way back through the facility. By a floral display you spy Saajret, apparently deep in conversation with Tiliia, the House Seineq engineer.`
			choice
				`	(Go speak with them.)`
				`	(Leave.)`
					goto end

> 			`	Tiliia spots you first, their skin breaking out in colors of excitement. "Saajret tells me of thy assent to Kaatrij's ranks," they say. "Excellent that thou might be a concerted ally to my own House as well."`

Reviewer: Was this intended to be ascent?
```suggestion
			`	Tiliia spots you first, their skin breaking out in colors of excitement. "Saajret tells me of thy ascent to Kaatrij's ranks," they say. "Excellent that thou might be a concerted ally to my own House as well."`
```

> 			`	Saajret hums and nods. "The captain is a canny sort," they say. "Capable of surviving our wiles, I believe." Tiliia laughs at that, Saajret continuing. "And Tiliia tells me thou were't of help to them as well, the reason by which our craft now gain an advantage through the wormhole."`

Reviewer: ```suggestion
			`	Saajret hums and nods. "The captain is a canny sort," they say. "Capable of surviving our wiles, I believe." Tiliia laughs at that, Saajret continuing. "And Tiliia tells me thou wert of help to them as well, the reason by which our craft now gain an advantage through the wormhole."`
```

	on complete
		conversation
			`	You land your ship on the outskirts of the same facility you visited with Sieasej in the past; it looks much the same, though now many of the green plants have turned various shades of yellow and orange as the seasons have changed. Saajret meets you in your ship's cockpit, now dressed in a slightly different black, well-pocketed outfit. They carry a device with a rotating drum in one of their arms, probably some sort of scanner.`

> 			`	"I shall keep in contact with thy craft," they say, "though I forsee no problems in my expedition. Expect me back about three hours hence."`

Reviewer 3: ```suggestion
			`	"I shall keep in contact with thy craft," they say, "though I foresee no problems in my expedition. Expect me back about three hours hence."`

```
